css/logo-nav.css

```diff
@@ -56,100 +56,250 @@ animation-delay: 0.7s;
 animation-delay: 1.1s;
 }
 
-.navbar-fixed-top .nav {
-    padding: 15px 0;
+/* //////////////////////////// */
+/*         NAVBAR CODE          */
+/* //////////////////////////// */
+
+.navbar-fixed-top .navbar-brand {
+    padding: 18px 0;
 }
 .navbar {
     background-color: white;
-    border-width: thin;
     -webkit-transition: .2s;
 }
-.navbar-gray {
-    background-color: rgb(248,248,248);
-}
-a {
-    color: black;
-}
-.navbar-fixed-top .navbar-brand {
-    margin-top: 5px;
-    padding: 0 15px;
-}
 .navbar-header .icon-bar {
-    background-color: black;
+    background-color: #E46E2E;
 }
-.navbar-active {
-    color: #E46E2E;
+.navbar-scroll .navbar-inner{
+    border-bottom: 1px solid #979797;
 }
-.navbar-scroll {
-    box-shadow: 1px 1px 1px #999; 
-    -moz-box-shadow: 1px 1px 1px #999;
-    -webkit-box-shadow: 1px 1px 1px #999;
-    animation-name: smooth;
-    animation-duration: 2s;
+.navbar-scroll .navbar-inner .dropdown-menu {
+    border-top: 1px solid #979797;
+}
+.nav > li:hover {
+    background-color: #f6f6f6
 }
 .nav > li > a {
-    padding: 10px 13px;
-    padding-top: 15px;
+    padding: 30px 0px 30px 0px;
+    color: #313940;
 }
 .nav > li > a:hover {
-    color: #E46E2E;
-    background-color: transparent;
-    -webkit-transition: .2s;
+    background-color: #f6f6f6;
+    color: black;
 }
 .nav > li > a:focus {
-    background-color: white;
+    background-color: #f6f6f6;
 }
-.nav > li > a:active {
-    background-color: #F8F8F8;
+.nav .open>a, .nav .open>a:focus {
+    background-color: #f6f6f6;
 }
-.nav > li > a:visited {
-    background-color: #F8F8F8;
+.nav li:hover {
+    background-color: #f6f6f6;
 }
-.img-container {
-    position: absolute;
-    width: 95%;
-    height: 100%;
-    text-align: center;
-    margin: 0 auto;
+.nav a {
+    width: 140px;
     
 }
-.img-container img {
-    vertical-align: center;
+.nav li{
+    padding: 0 0 0 0;
 }
-.header {
-    margin: 0 auto;
+#carrot {
+    margin-left: 1px;
+}
+.tab {
+    text-align: center;
+    font-weight: 500;
+    font-size: 16px;
+    color: #767474;
+}
+.tabs:hover a {
+    color:black;
+}
+.dropdown{
+    text-align: center;
+    font-weight: 500;
+    font-size: 16px;
+}
+.dropdown-menu {
+    min-width: 0px;
+    box-shadow: none;
```

Comment:
if you use box-shadow include -webkit and -moz prefixes as well (or never include prefixes anywhere)